docs/AssembleBody.md

```diff
@@ -25,3 +25,5 @@ These parts make up the box that serves as the rover's main body and equipment b
 ![Body right](images/Body05-Right.jpg)
 
 ![Body joined](images/Body06-Joined.jpg)
+
+The Rocker Body mounts should be spaced about 80mm from the edge of the front Body Corner.
```

Comment:
For proportion faithful to Curiosity rover side view seen in Onshape CAD file, there should be 62mm of aluminum extrusion visible between front face of "Rocker Body mount" and rear face of front "Body Corner".

In practice this can be moved forward/back as the builder pleases to maintain best balance for their rover chassis. Which will vary with every rover build as it depends on what's installed in equipment bay.

Reply:
Thank you. Updated the page to include your information on that. I know it's all 'builder specific', but having the info available as a baseline is useful for initial assembly.